back-end/app.js

```diff
@@ -1,4 +1,6 @@
 const express = require('express')
+const mongoose = require('mongoose')
```

Comment:
Could you make sure to include mongoose in the dependencies in package.json?

Reply:
Other than this, it looks good!

Reply:
done